config/jobs/kubernetes/sig-aws/eks/k8s-1.10-aws-eks-1.10.yaml

```diff
@@ -0,0 +1,73 @@
+
+presets:
+- env:
+  - name: AWSTESTER_EKS_TEST_MODE
+    value: "embedded"
+  - name: AWSTESTER_EKS_KUBERNETES_VERSION
+    value: "1.10"
+  - name: AWSTESTER_EKS_WAIT_BEFORE_DOWN
+    value: 1m0s
+  - name: AWSTESTER_EKS_DOWN
+    value: "true"
+  - name: AWSTESTER_EKS_ENABLE_WORKER_NODE_HA
+    value: "true"
+  - name: AWSTESTER_EKS_ENABLE_NODE_SSH
+    value: "true"
+  - name: AWSTESTER_EKS_ENABLE_LOG_ACCESS
+    value: "true"
+  - name: AWSTESTER_EKS_UPLOAD_TESTER_LOGS
+    value: "true"
+  - name: AWSTESTER_EKS_UPLOAD_WORKER_NODE_LOGS
+    value: "true"
+  - name: AWSTESTER_EKS_WORKER_NODE_INSTANCE_TYPE
+    value: m3.xlarge
+  - name: AWSTESTER_EKS_WORKER_NODE_ASG_MIN
+    value: "1"
+  - name: AWSTESTER_EKS_WORKER_NODE_ASG_MAX
+    value: "1"
+  - name: AWSTESTER_EKS_LOG_DEBUG
+    value: "false"
+  - name: AWSTESTER_EKS_LOG_ACCESS
+    value: "true"
+  - name: AWSTESTER_EKS_ALB_ENABLE
+    value: "false"
+  - name: AWS_SHARED_CREDENTIALS_FILE
+    value: /etc/eks-aws-credentials/eks-aws-credentials
+  labels:
+    preset-ci-kubernetes-e2e-1-10-aws-eks-1-10: "true"
+  volumeMounts:
+  - mountPath: /etc/eks-aws-credentials
+    name: eks-aws-credentials
+    readOnly: true
+  volumes:
+  - name: eks-aws-credentials
+    secret:
+      secretName: eks-aws-credentials
+
+periodics:
+# Run Kubernetes 1.10 branch e2e tests with EKS prod build 1.10
+# similar to "pull-kubernetes-e2e-gke" but as a periodic job
+- interval: 1h
+  name: ci-kubernetes-e2e-1-10-aws-eks-1-10-prod
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-ci-kubernetes-e2e-1-10-aws-eks-1-10: "true"
+
+  spec:
+    containers:
+    - image: 607362164682.dkr.ecr.us-west-2.amazonaws.com/awstester-e2e:20181029-141d7d02b498
```

Comment:
is this image public?

Reply:
This will be accessible with the credential that's used here. We can make it public even that's required :)

Reply:
Here's the Dockerfile for that image https://github.com/aws/awstester/blob/master/scripts/awstester-e2e.Dockerfile.

Reply:
or we can try and see :-)

Reply:
@krzyzacy Just listed it as a public image. Let's try! Thanks!